packages/svg.lua

```diff
@@ -3,6 +3,7 @@ local otparser = require("core/opentype-parser")
 
 local _drawSVG = function (svgdata, width, height, density, drop)
   local svgfigure, svgwidth, svgheight = svg.svg_to_ps(svgdata, density)
+  SU.debug("svg", string.format("PS: %s\n", svgfigure))
```

Comment:
```suggestion
  SU.debug("svg", function() return string.format("PS: %s\n", svgfigure) end)
```

This small hack is *almost* as silly as it looks. I ran some benchmarks a while back and discovered that one thing Lua does is evaluate all the expressions in these debug lines even if it doesn't use the result (i.e. the debug flag isn't active). This can cause significant overhead in some cases: ⓐ when the place being debugged is iterated over a lot or ⓑ the data being processed is large. I believe this falls into the latter category.

By passing a lambda function reference here the Lua interpreter is actually convinced not to bother evaluating the expression inside it unless the output is actually going to be used.